stests/cli/cache/list_run_deploys.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+
+from beautifultable import BeautifulTable
+
+from stests.cli.utils import get_table
+from stests.core import cache
+from stests.core.utils import args_validator
+from stests.core.utils import factory
+from stests.core.utils import logger
+from stests.core.orchestration import ExecutionAspect
+
+
+
+# CLI argument parser.
+ARGS = argparse.ArgumentParser("Displays summary information for a run.")
+
+# CLI argument: network name.
+ARGS.add_argument(
+    "network",
+    help="Network name {type}{id}, e.g. lrt1.",
+    type=args_validator.validate_network
+    )
+
+# CLI argument: run type.
+ARGS.add_argument(
+    "run_type",
+    help="Generator type - e.g. wg-100.",
+    type=args_validator.validate_run_type,
+    )
+
+# CLI argument: run index.
+ARGS.add_argument(
+    "run_index",
+    help="Run identifier, e.g. 1-100.",
+    type=args_validator.validate_run_index,
+    )
+
+
+
+def main(args):
+    """Entry point.
+    
+    :param args: Parsed CLI arguments.
+
+    """
+    # Pull data.
+    network_id = factory.create_network_id(args.network)
+    data = cache.state.get_deploys(network_id, args.run_type, args.run_index)
+    if not data:
+        logger.log("No run deploys found.")
+        return
+
+    # Set cols/rows.
+    cols = ["Deploy Hash", "Type", "Status", "Node"]
+    rows = map(lambda i: [
+        i.deploy_hash,      
+        i.typeof.name,
+        i.status.name,      
```

Comment:
What is the `"Status"` here? Is it the status from the `DeployInfo` object returned by client's `showDeploy`? That's what would be very useful.

Reply:
This comes from an internal enum:

class DeployStatus(enum.Flag):
    """Flag over set of deploy states.
    
    """
    NULL = enum.auto()
    DISPATCHED = enum.auto()
    FINALIZED = enum.auto()
    REJECTED = enum.auto()

This should be extended to be in sync with the DeployInfo object returned by the client's showDeploy.  Also we should store for each deploy the full contents of the DeployInfo object in the dB.